src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -57,9 +61,9 @@ public function resolve(Request $request, ArgumentMetadata $argument): array
         $message = '';
         if (null !== $options->expr) {
             if (null === $object = $this->findViaExpression($manager, $request, $options)) {
-                $message = \sprintf(' The expression "%s" returned null.', $options->expr);
+                $message = sprintf(' The expression "%s" returned null.', $options->expr);
```

Comment:
Backslash in front of all sprintf were added intentionally by @derrabus lately for optimization, they should be leaved as is

Reply:
yes sure... I will tell that to my IDE witch remove all ^^

Reply:
and question, why not make as in doctrine with a use ?

Reply:
That's just a matter of code style, Symfony codebase nearly never use "use function" (except from a few cases). I don't know if there's another reason, it's just the way it is 😄